osu.Game.Tests/Visual/UserInterface/TestSceneHistoryTextBox.cs

```diff
@@ -0,0 +1,173 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+#nullable disable
```

Comment:
Please don't `#nullable disable` in new code.

Reply:
Removed and instead initialized fields as `null!`.